client/src/components/ManageAccount/ManageAccount.jsx

```diff
@@ -91,7 +91,7 @@ const ManageAccountView = () => {
         )
       }
       contentClassName={`manage-account-content ${styles.panels}`}
-      contentLayoutName={isLoading ? `oneColumn` : `multiColumn`}
+      contentLayoutName={isLoading ? `oneColumn` : `multiColumnUnequal`}
```

Comment:
Let's revert for now. There is still a bug (on windows with short wide windows), and the bug looks better with `multiColumn` than it does with `multiColumnUnequal`.

```suggestion
      contentLayoutName={isLoading ? `oneColumn` : `multiColumn `}
```